src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -61355,6 +61355,197 @@ void F(C x)
                 );
         }
 
+        [Fact]
+        public void OtherComparisonsAsPureNullTests_01()
```

Comment:
Could you also test the `object` pattern in nested positions? `{ Property: object }` for example

Reply:
`object` is not a pattern in C# 8.0.  It is a pattern in C# 9.0.

---
In reply to: [365377713](https://github.com/dotnet/roslyn/pull/40892#discussion_r365377713) [](ancestors = 365377713)